src/main/java/mmosii/bookstore/controller/BookController.java

```diff
@@ -18,40 +21,48 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @RequiredArgsConstructor
 @RestController
 @RequestMapping(value = "api/books")
 public class BookController {
     private final BookService bookService;
 
     @GetMapping
-    public List<BookDto> findAll() {
-        return bookService.findAll();
+    @Operation(summary = "Get all books", description = "Get a list of all available books")
+    public List<BookDto> findAll(Pageable pageable) {
```

Comment:
you could also use `@PageableDefault` in case client didn't provide pagination parameters